sdk/identity/azure-identity-broker/README.md

```diff
@@ -23,9 +23,9 @@ pip install azure-identity-broker
 
 This package enables broker support via `InteractiveBrowserBrokerCredential`, which is a subclass of the Azure Identity library's [InteractiveBrowserCredential][ibc].
 
-### Parent window handles
+### Parent window handle
 
-When authenticating interactively via `InteractiveBrowserBrokerCredential`, a parent window handle is required to ensure that the authentication dialog is shown correctly over the requesting window. In the context of graphical user interfaces on devices, a window handle is a unique identifier that the operating system assigns to each window. For the Windows operating system, this handle is an integer value that serves as a reference to a specific window.
+When authenticating interactively via `InteractiveBrowserBrokerCredential`, a parent window handle is required to ensure that the authentication dialog is shown correctly over the requesting window. In the context of graphical user interfaces on devices, a window handle is a unique identifier that the operating system assigns to each window. For the Windows operating system, this handle is an integer value that serves as a reference to a specific window. On macOS, it is an integer-based identifier.
```

Comment:
Tbh, I don't really know how to get the parent window handle on macOS for GUI-based apps. All my macOS testing has been done passing in `msal.PublicClientApplication.CONSOLE_WINDOW_HANDLE`. 

I think we should also update the heading for the example we have to specify that it is for WIndows.

```
### Authenticate with `InteractiveBrowserCredential` on Windows
```

Then, perhaps consider having an example for authenticating a console app which would be something Mac users would use.

````python
### Authenticate a console app on Windows or macOS

```
from azure.identity.broker import InteractiveBrowserBrokerCredential
from azure.storage.blob import BlobServiceClient
import msal

credential = InteractiveBrowserBrokerCredential(
    parent_window_handle=msal.PublicClientApplication.CONSOLE_WINDOW_HANDLE
)
client = BlobServiceClient(account_url, credential=credential)
```
````

We should maybe consider exposing the CONSOLE_WINDOW_HANDLE property as an API under the azure-identity-broker surface area at some point.

Reply:
When we say "integer-based identifier", what does the "identifier" represent? What does it identify? That question should be answered in the docs.

Reply:
But we mark window handle required (even on macOS). We cannot only tell user how to use it on windows, right?